src/MasterSite.php

```diff
@@ -304,6 +304,56 @@ function ($tests) {
             }
         );
 
+        // Maximum revisions to keep whenever the editor save a post
+        add_filter(
+            'wp_revisions_to_keep',
+            function ($revisions) {
+                $revisions_to_keep = get_option('revisions_to_keep');
+                return $revisions_to_keep ?: $revisions;
+            },
+            10,
+            2
+        );
+
+        // Registering into the Settings > Writing setting page
+        add_action(
+            'admin_menu',
+            function (): void {
```

Comment:
We can move this function code into a new function , as [like](https://github.com/greenpeace/planet4-master-theme/blob/f3d2d05e187ef887d14614b149b6e6a8817cf254/src/MasterSite.php#L125) -
`add_action('admin_enqueue_scripts', [$this, 'enqueue_admin_assets']);`

Reply:
That sounds logic!

Reply:
I mean to say -
we can make it
`add_action(‘admin_menu’, [$this, ‘add_post_revisions_setting’]); `
and we will add a new function with -
```
public function add_post_revisions_setting() {
...
}
```
Sorry, if I confuse you earlier